agent/util-scripts/unittests

```diff
@@ -118,13 +118,15 @@ function _run {
         exit 1
     fi
 
+    export _PBENCH_UNIT_TEST_NAME=${tname}
     local opts=${options[${tname}]}
     echo "${tname} ${tscrpt} ${opts}"
     echo "+++ Running ${tname} ${tscrpt} ${opts}" >> ${_testout}
 
     _runout=${_testroot}/run.out
     eval ${tscrpt} ${opts} > ${_runout} 2>&1
```

Comment:
Rather than adding both an `export` a line 121, and an `unset` at line 129, can we consider using an inline environment variable, e.g. `_PBENCH_UNIT_TEST_NAME=${tname} eval ${tscrpt} ${opts} > ${_runout} 2>&1`?